src/module-loader/dotnet/src/MorganStanley.ComposeUI.ModuleLoader.Abstractions/ModuleManifest.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Morgan Stanley makes this available to you under the Apache License,
+// Version 2.0 (the "License"). You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0.
+// 
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership. Unless required by applicable law or agreed
+// to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions
+// and limitations under the License.
+
+namespace MorganStanley.ComposeUI.ModuleLoader;
+
+public abstract class ModuleManifest
+{
+    public string Name { get; }
```

Comment:
I think we talked about having an `Id` (that's unique and contains the version) and `Name` (which is the 'friendly name' of the module, without version). Startup and running instance information should refer to the Id because the catalog can contain multiple versions of the same module. A module can be started by its `Id`, and then identified by its `InstanceId` (because multiple instances of the same module can be started).